google_symptoms/delphi_google_symptoms/patch.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module is used for patching data in the delphi_google_symptom package.
+
+To use this module, you need to specify the range of issue dates in params.json, like so:
+
+{
+  "common": {
+    ...
+  },
+  "validation": {
+    ...
+  },
+  "patch": {
+    "patch_dir": "/Users/minhkhuele/Desktop/delphi/covidcast-indicators/doctor_visits/AprilPatch",
+    "start_issue": "2024-04-20",
+    "end_issue": "2024-04-21"
+  }
+}
+
+It will generate data for that range of issue dates, and store them in batch issue format:
+[name-of-patch]/issue_[issue-date]/doctor-visits/actual_data_file.csv
```

Comment:
```suggestion
[params patch_dir]/issue_[issue-date]/doctor-visits/actual_data_file.csv
```

Reply:
I copied this directly from mihn's doctor visit should I also change it there too?

Reply:
If it still makes sense in that context, then yea sounds good.